ca.vlacroix.Tally.json

```diff
@@ -0,0 +1,38 @@
+{
+	"app-id": "ca.vlacroix.Tally",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "47",
+	"sdk": "org.gnome.Sdk",
+	"command": "tally",
+	"finish-args": [
+		"--device=dri",
+		"--share=ipc",
+		"--socket=fallback-x11",
+		"--socket=wayland"
+	],
+	"modules": [
+		"shared-modules/lua5.4/lua-5.4.json",
+		{
+			"name": "lgi",
+			"no-autogen": true,
+			"make-args": ["-C", "lgi", "LUA_VERSION=5.4"],
```

Comment:
is the makefile in `lgi` directory ? Then you need `subdir: lgi`

Reply:
Helpful again. Thank you!